queries/staking/useSNXLockedValueQuery.ts

```diff
@@ -46,17 +47,19 @@ const useSNXLockedValueQuery = (options?: QueryConfig<number>) => {
 			let snxLocked = 0;
 
 			for (const { collateral, debtEntryAtIndex, initialDebtOwnership } of holders) {
+				const formattedCollateral = Number(synthetix.js?.utils.formatEther(collateral));
```

Comment:
could we have a different name for `formattedCollateral` because it hints it might be formatted like a string (e.g - `1000` becomes `1,000.00`)